dist/backdraft.js

```diff
@@ -1303,6 +1303,19 @@ _.extend(Plugin.factory, {
       }
     },
 
+    // takes a hash of { columnTitle: columnState, ... }
+    setColumnVisibilities: function(columns) {
+      _.each(columns, function(state, title) {
+        if (!state && this._columnManager.columnConfigForTitle(title).required) {
+          throw new Error("can not disable visibility when column is required");
+        }
+      }, this);
+      this._columnManager.visibility.set(columns);
```

Comment:
You're right, k yeah that is another level of control.  And after your earlier comment I looked further into that `this._columnManager.visibility.set(columns)` interface (and how we also call `.set(title, state)` else where.. it's a confusing interface in that it accepts bulk or just one, and is okay with going from a big hash to just setting one (therefore blowing away the other values) as I think that Backbone model is only used transiently to fire change events ... so yeah, like you said this allows just one change event to fire versus 1 for each.. 

But it bugs me that the check & raise is the same as the method just above it, and this each loop down below is the same too